dist/grep_handler.py

```diff
@@ -0,0 +1,25 @@
+import os
+import re
+import sys
+
+from sfs import *
```

Comment:
Stylistic thing: I want us to support `from sfs import *` but when thinking about what makes for easily understandable examples I think it's better to say `import sfs` and then have `sfs.open()`, `sfs.mount()`, etc. in the code.